src/modules/site-v2/base/views/tools/genetic_mapping.py

```diff
@@ -94,7 +136,7 @@ def report(id):
   fluid_container = True
   data_url = generate_blob_url(mapping.get_bucket_name(), mapping.get_data_blob_path())
   # if hasattr(mapping, 'mapping_report_url'):
-  if hasattr(mapping, 'report_path'):
+  if mapping.report_path is not None:

```

Comment:
Handle mapping not found when user submits an invalid <id>

Reply:
Fixed in 0f531415cf290a2cc0ed24dd25e74b47b132c058 -- see also discussion at https://github.com/AndersenLab/CAENDR/pull/145#discussion_r1132691789